install_nim.sh

```diff
@@ -57,6 +57,11 @@ while ((0 < $#)); do
   esac
 done
 
+# get exact version of stable
+if [[ "$nim_version" = "stable" ]]; then
+    nim_version=$(curl -sSL http://nim-lang.org/channels/stable)
```

Comment:
Please use `https`, not `http`.

```suggestion
    nim_version=$(curl -sSL https://nim-lang.org/channels/stable)
```